src/languages/asciidoc.js

```diff
@@ -107,17 +107,38 @@ export default function(hljs) {
         begin: '^(NOTE|TIP|IMPORTANT|WARNING|CAUTION):\\s+',
         relevance: 10
       },
-      // inline strong
+      // inline unconstrained strong (single line)
       {
         className: 'strong',
-        // must not follow a word character or be followed by an asterisk or space
-        begin: '\\B\\*(?![\\*\\s])',
-        end: '(\\n{2}|\\*)',
-        // allow escaped asterisk followed by word char
-        contains: [{
-          begin: '\\\\*\\w',
-          relevance: 0
-        }]
+        begin: /\*\*(.+?)\*\*/
+      },
+      // inline unconstrained strong (multi-line)
+      {
+        className: 'strong',
+        begin: /\*{2}(\s\S+)*\*{2}/,
```

Comment:
This of course as the same issues with begin/end... we need access to `contains`  so we can properly highlight markup that appears inside of bold.